prog/github/github_repository_nexus.rb

```diff
@@ -80,6 +80,25 @@ def check_queued_jobs
     @polling_interval = (remaining_quota < 0.5) ? 15 * 60 : 5 * 60
   end
 
+  def check_cache_usage
```

Comment:
Nit: Name of the function can be updated as we are not checking the cache usage but also destroy it if necessary. `cleanup_cache` might be a good candidate

Reply:
Makes sense. I renamed it